src/phrases/phrases-nb.ts

```diff
@@ -1135,6 +1135,7 @@ const phrases = {
         show: 'Vis byline',
         hide: 'Skjul byline',
       },
+      isDecorative: 'Bildet er dekorativt',
```

Comment:
Usikker på kva som er mest beskrivende her. Kanskje heller "Bildet er ment dekorativt". Ref beskrivelse på https://trello.com/c/4PGe8S2i/192-ed-m%C3%A5-gi-mulighet-for-tom-alt-tekst-p%C3%A5-dekorative-bilder-i-artikkel

Reply:
@gunnarvelle Jeg endrer til orginalforslag: "bildet er kun dekorativt i denne sammenhengen"
Det fikk meg til å huske at det er kun i artikkel-sammenheng man skal kunne få dette opp. Da slenger jeg på en sjekk på om det er i modal.

Reply:
on second thought blir det vel _kun_ i artikkelsammenheng, så lager et ny prop